beacon-chain/blockchain/service.go

```diff
@@ -125,11 +125,6 @@ func NewService(ctx context.Context, cfg *Config) (*Service, error) {
 
 // Start a blockchain service's main event loop.
 func (s *Service) Start() {
-	beaconState, err := s.beaconDB.HeadState(s.ctx)
```

Comment:
Note: `beaconState` used to be nil here most of the time. 

I took this out and removed the if `beaconState == nil` below.